lerna.json

```diff
@@ -6,11 +6,6 @@
   ],
   "version": "1.0.0-prerelease.29",
   "ignoreChanges": [
-    "elements/*/*.css",
-    "elements/*/*.js",
-    "elements/*/*.map",
-    "elements/*/dist/*.css",
-    "elements/*/dist/*.js",
-    "elements/*/dist/*.map"
+    "elements/*/dist/**/*"
```

Comment:
Would it make sense to ignore the entire dist directory instead of just the files nested inside it?  Also, there aren't any folders in the dist directory, just files so if you did want to keep the dist but ignore the files in it, `dist/*` would be sufficient.

Reply:
I think we're being extra careful here on purpose.